### PR TITLE
[docs] Doc changes in support for embeddings

### PR DESCRIPTION
**Pull request by pablo-gar**
Notes for reviewers:

1. @atarashansky please review notebooks you authored. I simplified the code of the bio explorations, fixed some issues, and did some small editorial changes, please review it thoroughly and make sure findings still hold.
2. @atarashansky please correct leiden error on scvi prediction notebook.
3. @bkmartinjr please review technical clarity of the notebook you authored, I did major organizational changes.
4. @bkmartinjr please review technical clarity of the notebooks:
   5. `api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb` @mlin please review this one as well.
   6. `api/python/notebooks/api_demo/census_access_maintained_embeddings.ipynb` 
7. @mlin please review the R changes and 

All of you please review:
 - `docs/cellxgene_census_docsite_landing.md`
 - `docs/examples.rst`

----------

Preview of Python tutorials updated section

image

### CODE REVIEW

**Review by bkmartinjr — CHANGES_REQUESTED** [review]
Noted:
1. Notebooks document which tiledbsoma version they require, but not which cellxgene-census version they require.  Recommend simply saying "cellxgene-census package version 1.9.1 or later", as that covers both the metadata tag name change _and_ tiledbsoma.
2. census_embedding.ipynb should document behavior of `get_embedding` when there is no embedding for a cell (ie. it inserts NaN)
3. census_access_maintained_embeddings does not document what happens with missing embeddings. Or how to get the same behavior as `get_embedding` (NaN for missing).  I suspect the code currently puts _zero_ into missing embeddings which may be confusing.


Very minor stuff:
1. I would name the embedding notebooks with a consistent scheme so they are easier to identify. E.g., census_embedding_....ipynb
2. It wouldn't hurt to add a reminder about using the python `help()` function to see docstrings, e.g., `help(get_embedding)`
3. The census_access_maintained_embeddings notebook notes that there is partial support for mouse, but the census_embedding_access notebook does not do the same.  Both should probably make a general statement like "embeddings may be available for one or both organisms - see the Models web page for the latest availability"


Checks I did:
* census_embedding.ipynb - otherwise looks good
* census_access_maintained_embeddings.ipynb - otherwise looks good.

**Comment by pablo-gar** [comment]
@bkmartinjr 

> Noted:
> 1. Notebooks document which tiledbsoma version they require, but not which cellxgene-census version they require.  Recommend simply saying "cellxgene-census package version 1.9.1 or later", as that covers both the metadata tag name change _and_ tiledbsoma.

Done

>3. census_embedding.ipynb should document behavior of `get_embedding` when there is no embedding for a cell (ie. it inserts NaN)

Done

> 4. census_access_maintained_embeddings does not document what happens with missing embeddings. Or how to get the same behavior as `get_embedding` (NaN for missing).  I suspect the code currently puts _zero_ into missing embeddings which may be confusing.

Not going to add notes since there are no missing values for those embeddings.

> Very minor stuff:
> 1. I would name the embedding notebooks with a consistent scheme so they are easier to identify. E.g., census_embedding_....ipynb

We have committed these names for the webpage, I don't want to request change to Seve on that for now since his plate is pretty full

> 2. It wouldn't hurt to add a reminder about using the python `help()` function to see docstrings, e.g., `help(get_embedding)`

Done

> 3. The census_access_maintained_embeddings notebook notes that there is partial support for mouse, but the census_embedding_access notebook does not do the same.  Both should probably make a general statement like "embeddings may be available for one or both organisms - see the Models web page for the latest availability"

Done

File: docs/cellxgene_census_docsite_landing.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<span style="background-color: #F6DDCC; color; font-size: 18px"> 🚨 A **normalized layer**, plus cell and gene **pre-calculated stats** are now available!  Find out more [here.](articles/2023/20231012-normalized_layer_precalc_stats.md)</span>
+<span style="background-color: #f3bfcb; color; font-size: 18px"> 🚀 New to the Census: we’ve created a **centralized hub of models and embeddings** using Census data. [Check it out](https://cellxgene.cziscience.com/census-models)!
+
+ </span>
```
Comment (bkmartinjr) [inline docs/cellxgene_census_docsite_landing.md:3]: the image below is quite dated wrt statistics for the Dec LTS release. update?

Comment (pablo-gar) [reply docs/cellxgene_census_docsite_landing.md:3]: good point reminds me I need to update https://chanzuckerberg.github.io/cellxgene-census/cellxgene_census_docsite_data_release_info.html

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 2]
```diff
@@ -0,0 +1,1003 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7ff430af-04a0-425b-bdf4-ed0a3aa571b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!aws s3 sync s3://cellxgene-contrib-public/models/geneformer/2023-10-23/homo_sapiens/fined-tuned-model/ ./fine_tuned_geneformer  --no-progress --only-show-errors"
```
Comment (bkmartinjr) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:130]: wrong link

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 3]
```diff
@@ -0,0 +1,1003 @@
...
...AmTZqG2Xz4bHOH9C7oOliBVyGEEF8ufec3CCGE+FJbXOvBGUzUGzLoYG1JZKE7NGnN61BZPULlxQOvM2n0PPaUdE8/M+bkMP/0qxnjzWfGju4sbXGvl/YV7/a5VqEjn9L0ElRFxTZyVDI1N0DIaiFsTZw7Ho8TiUQACDgcbM3y4DekfkVZgonMdRUV+5KJGnoym82cddZiTj99Eeec8xWmTBm8QGtvM2fOw+VKTCR0udKZMePIaiAJIYT44pORIiGEGGLMipG+ibEhYFF4+fh0mjxGoqZEQOK2pjP+5p+zf9MKihQ3RVPnUd1YT3qHHV1RUHqs/1GMRuLxOJ2dfmw2O6Ze64AshUUUfu8WvO8sQzMY2GwEXe3/2ZyiKKSnu2lsrE9uK6iuoy0rA2+GB7XHcfF4PFkzyGg0kpdX8JHeF6fTxXnnXUIkEj7idUhCCCG+HCQoEkKIIcZz2un4131I3N+Rst0e1tEKconGEhnXVEUlw+Lm/zY9SEfEj0k1cW2wBCUQAUWhYkQppXv3owCmwkIMM2bwwgtP4/d3YLFYOL5oBNrqVShGA5nnXUA0I4N39u7A57SRn1+Ae/9+IppGzNw3iYKu6zTX1ZDX0EjIZiO9tY3c+gYiFjPFC04mLc3BwYO1LFv2GvF4DIPBwPz5JzNsWOnHfn8kIBJCiKFH0fUej/k+R/G4Rmtr5+fdDSGEGBJiPh+hA/uJdXSw78WniIfDbBhjo3ZSEb6wj5geJ9PqwWFyUNlRnTyuIC2P7479Oi+++B9isSjmUIgcRzoLLvwq769ZwYEDewGwBoJMWr8JVUtM01MdDvaffCL13tbkufJr6mjIz0XrGuXpz+QP1mMLdieBMI0aRayyEmNGBptKCmjvse5HURQuvvgKLBZZC/RZychIw2CQmfhCiC8+GSkSQoghyJiejmPKVCrbq3ko6gAciR2h7qClJdRGS6gt5bi4ruFyuTnrrMXs2red9nArZTELkcpKotFosl2avzMZEAFofj/RxkYwd3/thC0WPC1ttORkDdjPnRPGMb09gDkex5juxt9V5DVaX0+Z18vGWdOSbXVdx+/3DxgUHTiwl9raKhwOFxMnTk1OuRNCCCEkKBJCiCHMpB6+/s8hqqKycHgi61vcqrOk8y188U6MMZ2F//ShjelOTBBIs6OrKkpXYBQzGOjoNaDgaWkhs6kFd2
```
Comment (mlin) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:751]: We did switch to `emb_layer=0` right?

Comment (pablo-gar) [reply api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:750]: good catch! No wonder why the data projection did not look good anymore

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb (hunk 2 above)
Comment (pablo-gar) [reply api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb]: corrected

File: docs/cellxgene_census_docsite_landing.md (hunk 1 above)
Comment (pablo-gar) [reply docs/cellxgene_census_docsite_landing.md:3]: done

**Review by bkmartinjr — APPROVED** [review]

**Pull request merged**